libsolidity/interface/StandardCompiler.cpp

```diff
@@ -708,15 +704,14 @@ std::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompiler:
 			{
 				/// If the import succeeded, let mark all the others as warnings, otherwise all of them are errors.
 				ret.errors.append(formatError(
-					found ? Error::Severity::Warning : Error::Severity::Error,
-					"IOError",
+					Error::Type::IOError,
```

Comment:
Hi, so, after taking help from @matheusaaguiar  , a command line test has been added to possibly cover the case we are mentioning here. 

But it seems to be passing. Perhaps you could take a look and see if we missed anything on the test case ?

Reply:
> But it seems to be passing. Perhaps you could take a look and see if we missed anything on the test case ?

The test is passing because the expectations are wrong. According to your `output.json` you expect the input to produce 3 `IOError`s (`"severity":"error","type":"IOError"`). But that's not what should happen. Here's what 0.8.16 outputs when `contract.sol` does exist:

```json
{
    "errors":
    [
        {
            "component": "general",
            "formattedMessage": "Cannot import url (\"/projects/contracts.sol\"): File not found. Searched the following locations: \"\".",
            "message": "Cannot import url (\"/projects/contracts.sol\"): File not found. Searched the following locations: \"\".",
            "severity": "warning",
            "type": "IOError"
        },
    ]
    "sources":
    {
        "url_not_found.sol":
        {
            "id": 0
        }
    }
}
```
Your test should be passing with this as the expectation.

Reply:
Changed the test case to resolve this comment.